mat/cholesky_example_test.go

```diff
@@ -42,7 +42,8 @@ func ExampleCholesky() {
 	fmt.Printf("x = %0.4v\n", mat.Formatted(&x, mat.Prefix("    ")))
 
 	// Extract the factorization and check that it equals the original matrix.
-	t := chol.LTo(nil)
+	t := &mat.TriDense{}
```

Comment:
`var t mat.TriDense` and resultant changes (for consistency with the `mat.SymDense` use below.

Reply:
Done.